Update set-up-safe-links-policies.md (#10397)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/set-up-safe-links-policies.md

4. On the **Users and domains** page that appears, identify the internal recipients that the policy applies to (recipient conditions):
   - **Users**: The specified mailboxes, mail users, or mail contacts.
   - **Groups**:

>      - Members of the specified distribution groups(will support non-mail enabled security groups within distribution groups) or mail-enabled security groups.

Reviewer: ```suggestion
     - Members of the specified distribution groups (will support non-mail-enabled security groups within distribution groups) or mail-enabled security groups.
```

5. On the **URL & click protection settings** page that appears, configure the following settings:

>    - **Action on potentially malicious URLs within Emails(Email & Time of Click)** section:

Reviewer: ```suggestion
   - **Action on potentially malicious URLs within Emails (Email & Time of Click)** section:
```

     - **On: Safe Links checks a list of known, malicious links when users click links in email**: Select this option to turn on Safe Links protection for links in email messages. If you select this option, the following settings are available:

>        - **Apply Safe Links to email messages sent within the organization(Email – Intraorg & Time of Click)**: Select this option to apply the Safe Links policy to messages between internal senders and internal recipients. Turning this on will enable link wrapping for all intraorg messages.

Reviewer: ```suggestion
       - **Apply Safe Links to email messages sent within the organization (Email – Intraorg & Time of Click)**: Select this option to apply the Safe Links policy to messages between internal senders and internal recipients. Turning this on will enable link wrapping for all intraorg messages.
```

>        - **Apply real-time URL scanning for suspicious links and links that point to files(Email)**: Select this option to turn on real-time scanning of links in email messages from external senders. If you select this option, the following setting is available:
>          - **Wait for URL scanning to complete before delivering the message(Email)**: Select this option to wait for real-time URL scanning to complete before delivering the message from senders external . Recommended setting is "On".
>        - **Do not rewrite URLs, do checks via SafeLinks API only(Time of Click)**: Select this option to prevent URL wrapping and skip reputation check during mailflow. Safe Links is called exclusively via APIs at the time of URL click by Outlook clients that support it.

Reviewer: ```suggestion
       - **Apply real-time URL scanning for suspicious links and links that point to files (Email)**: Select this option to turn on real-time scanning of links in email messages from external senders. If you select this option, the following setting is available:
         - **Wait for URL scanning to complete before delivering the message (Email)**: Select this option to wait for real-time URL scanning to complete before delivering the message from external senders. The recommended setting is **On**.
       - **Do not rewrite URLs, do checks via SafeLinks API only (Time of Click)**: Select this option to prevent URL wrapping and skip reputation check during mail flow. Safe Links is called exclusively via APIs at the time of URL click by Outlook clients that support it.
```

        When you're finished, click **Done**.

>    - **Actions for potentially malicious URLs in Microsoft Teams(Time of Click)** section:

Reviewer: ```suggestion
   - **Actions for potentially malicious URLs in Microsoft Teams (Time of Click)** section:
```

     - **On: Safe Links checks a list of known, malicious links when users click links in Microsoft Teams**: Select this option to enable Safe Links protection for links in Teams. Note that this setting might take up to 24 hours to take effect.

     > [!NOTE]
     > Currently, Safe Links protection for Microsoft Teams is not available in Microsoft 365 GCC High or Microsoft 365 DoD.

>    - **Actions for potentially malicious URLs in Microsoft Office apps(Time of Click)** section:

Reviewer: ```suggestion
   - **Actions for potentially malicious URLs in Microsoft Office apps (Time of Click)** section:
```
